core/src/main/resources/geocat/Messages.properties

```diff
@@ -0,0 +1,9 @@
+# Geocat specific messages
+geocat_unpublish_email_subject=Geocat.ch notification of unpublished records
+geocat_unpublish_email_message=Hi {{userName}},<br>\
+<br>\
+At least one of your metadata records on Geocat.ch were automatically unpublished<br>\
+because they were found to be invalid.<br>\
+<br>\
+The following records were affected:<br>\
```

Comment:
Regards ?
invalid -> unpublish rather than unpublish because invalid ?

Reply:
I'll leave it like that since the customer will send us a final version of the email template.